static/app/components/globalDrawer/components.tsx

```diff
@@ -74,6 +79,7 @@ export function DrawerPanel({
           ref={mergeRefs(panelRef, ref)}
           transitionProps={transitionProps}
           panelWidth="var(--drawer-width)" // Initial width only
+          panelHeight={drawerHeight}
           className="drawer-panel"
```

Comment:
Thanks for clarifying about the bug, I was able to repro as well, good catch 🫡 . I think it might be worth swapping `drawerHeight` for `drawerCss` prop instead just we don't people can make other changes if they need for their use cases.

Also, I'm in favour of just adding height: 100% to the default drawer so this doesn't trip up any one else putting scrollable content in the drawers.

Reply:
I was thinking like `style={drawerCss}`, where `drawerCss` comes from 
```
import {css} from '@emotion/react';
```